swift-sdk/IterableAPI.swift

```diff
@@ -171,8 +171,8 @@ import UIKit
     @objc(updateUser:mergeNestedObjects:onSuccess:onFailure:)
     public static func updateUser(_ dataFields: [AnyHashable: Any],
                                   mergeNestedObjects: Bool,
-                                  onSuccess: OnSuccessHandler? = nil,
-                                  onFailure: OnFailureHandler? = nil) {
```

Comment:
We should keep `= nil` here because then Swift code can call without specifying both success and failure handlers.